docs/topics/vscode-extension-configuring-analysis.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * docs/vs-code-extension-guide/master.adoc
+
+[id='vscode-extension-configuring-analysis_{context}']
+= Configuring a run configuration
+
+You can configure multiple run configurations to run against each project.
+
+.Prerequisites
+
+* Projects imported into VS Code.
+
+.Procedure
+
+. In *Extensions* view, click the *{ProductName}* icon (image:vs_mta_ex_icon.png[MTA code extension]) on the Activity bar.
+. Click the *+* (plus sign) next to *{ProductName}* to add a run configuration.
+. Complete the following configuration fields:
+
+* *Input*: Click *Add* and either:
+
+** Enter the input file or directory and press Enter.
+** Click *Open File Explorer* and select the directory.
+
+* *Target*: Select one or more target migration paths.
+
+. Optional: Configure any advanced options needed.
```

Comment:
This is pretty vague. Can you give an example of such "advanced options"? Or can you eliminate the step?